docs/DeveloperGuide.adoc

```diff
@@ -347,8 +618,10 @@ _{More to be added}_
 == Non Functional Requirements
```

Comment:
Consider adding the following NFRs 🤔 
- Able to have 1000 flashcards without sluggish performance
- Flashcards should work without Internet connectivity